newton/_src/geometry/contact_reduction_hydroelastic.py

```
@@ -216,7 +597,14 @@ def reduce_hydroelastic_contacts_kernel(
             bin_id = get_slot(normal)
             key = make_contact_key(shape_a, shape_b, bin_id)
 
-            entry_idx = hashtable_find_or_insert(key, reducer_data.ht_keys, reducer_data.ht_active_slots)
+            entry_idx = int(-1)
```

Ruff (0.15.21)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)